docs/en/dataset_prepare.md

```diff
@@ -376,3 +383,16 @@ python tools/convert_datasets/isaid.py /path/to/iSAID
 ```
 
 In our default setting (`patch_width`=896, `patch_height`=896,　`overlap_area`=384), it will generate 33978 images for training and 11644 images for validation.
+
+### EGOHANDS
+
+The training and validation set of EGOHANDS could be download from [here](https://staffnet.kingston.ac.uk/~ku15565/CHASE_DB1/assets/CHASEDB1.zip).
```

Comment:
please pay attention to this URL, it might not be for egohands?

Reply:
you're right. Just corrected it.